test-environment.lua

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env -S luajit
 require"lib.ctx"(require"config", function(ctx)
 local U = require"lib.util"
-local Z = ctx.interfaces.zigbee[1]
+local _ = ctx.interfaces.zigbee[1]
```

Comment:
can be dropped altogether, I guess. But OTOH, the whole file does not make a lot of sense.

Reply:
Done.